File: src/main/java/NekoEssentials/commands/fly.java
package NekoEssentials.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import static NekoEssentials.Main.c;
import static NekoEssentials.Main.config;

public class fly implements CommandExecutor {

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (!label.equalsIgnoreCase("fly") || !(sender instanceof Player)) {
            return false;
        }

        Player player = (Player) sender;

        if (!player.hasPermission("NekoEssentials.fly")) {
            player.sendMessage(c(config.getConfig().getString("no-permission")));
            return true;
        }

        if (player.getAllowFlight()) {
            player.setAllowFlight(false);
            player.sendMessage(c(config.getConfig().getString("fly-off")));
        } else {
            player.setAllowFlight(true);
            player.sendMessage(c(config.getConfig().getString("fly-on")));
        }

        return true;
    }
}

File: src/main/java/NekoUtility/commands/fly.kt
package NekoUtility.commands

import NekoUtility.Main
import NekoUtility.Main.Companion.c
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class fly : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender !is Player) {
            return false
        }
        val p = sender
        if (!p.hasPermission("nekoutility.fly")) {
            p.sendMessage(c(Main.config!!.getConfig().getString("no-permission")))
            return true
        }
        if (p.allowFlight) {
            p.allowFlight = false
            p.sendMessage(c(Main.config!!.getConfig().getString("fly-off")))
        } else {
            p.allowFlight = true
            p.sendMessage(c(Main.config!!.getConfig().getString("fly-on")))
        }
        return true
    }
}